dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java

```diff
@@ -178,6 +213,20 @@ else if (pathInfo.equals("/")) {
 								}
 								httpRoute = null;
 							}
+						} else if ((pathInfo == null) && 
+								(webAppDispatcherContext40.getServletPathForMapping() != null) 
+								&& (!webAppDispatcherContext40.getServletPathForMapping().isBlank())){
```

Comment:
Is `isBlank()` what you want here?  It does not mean is empty string necessarily.

Reply:
fixed

Reply:
What do you mean by fixed?  It is unchanged.  Are you saying the answer to my question is `yes`?

Reply:
Sorry, I intended to change it to `isEmpty()` . I had it changed in an earlier branch,  but some weird rebasing and chery-picking made me do _another_ branch and then I forgot about it. Updated with new commit.

`isEmpty` was used everywhere else, not sure why I had `isBlank`.